README.md

```diff
@@ -1,3 +1,51 @@
-# music-of-madagascar
-A list of artists and genres from Madagascar
+# Music of Madagascar
 
+A list of artists and genres from Madagascar.
+See: [onja-madagascar.github.io/music-of-madagascar](https://onja-madagascar.github.io/music-of-madagascar/)
+
+## About
+
+This repo was started as a school project at [Onja.org](https://onja.org), as we love Malagasy music.
+
+## How to contribute
```

Comment:
It is not that clear so you should make some detail in it.

Reply:
Ah, my headings look wrong. One can contribute by developing, or adding artists, or adding genres

Reply:
Thanks, will fix the heading for development